ceph-iscsi.spec

```diff
@@ -68,7 +68,8 @@ Requires:       python3-configshell
 %endif
 %endif
 
-BuildRequires:  systemd
+BuildRequires: systemd-rpm-macros
+%{?systemd_requires}
```

Comment:
@ricardoasmarques 

Will this be ok for SUSE? I saw you have similar scriplets here

https://en.opensuse.org/openSUSE:Systemd_packaging_guidelines

but in "(Build) Requirements" it looks like you also have to check for a specific suse version. Is that check needed or ceph-iscsi only in the newer version?

Reply:
@smithfarm can you help here? thanks.

Reply:
@mikechristie The non-systemd versions of openSUSE are all EOL now, so that document is outdated. In other words, this line is OK for SUSE :-)